src/server/routes/revision.js

```diff
@@ -237,14 +238,25 @@ router.get('/:id', (req, res, next) => {
 					formatWorkChange
 				)
 			);
-			const props = {revision: revision.toJSON(), diffs};
-			res.render('page', {
+			const props = Object.assign({}, req.app.locals, res.locals, {
+				revision: revision.toJSON(),
 				title: 'RevisionPage',
-				markup: ReactDOMServer.renderToString(RevisionPage(props))
+				diffs
 			});
+			const markup = ReactDOMServer.renderToString(
+				<Layout {...props}>
+					<RevisionPage />
```

Comment:
I think it's worth making the change. As I understand it, the difference in the props between this rendering and the rendering in the controller will make react unable to correctly link the two and lose us most of the benefit of isomorphic react, as it will have to re-render the page from scratch.